tests/phpunit/api/v3/CountryTest.php

```diff
@@ -104,4 +104,34 @@ public function testCreateDuplicateFail() {
     $this->assertEquals(1, $check);
   }
 
+  /**
+   * Test that the list of states is in the correct format when chaining
+   * and using sequential.
+   */
+  public function testCountryStateChainSequential() {
+    $result = $this->callAPISuccess('Country', 'getsingle', [
+      'iso_code' => 'US',
+      'api.Address.getoptions' => [
+        'field' => 'state_province_id',
+        'country_id' => '$value.id',
+        'sequential' => 1,
```

Comment:
@demeritcowboy you could have left sequential off here just to demonstrate that sequential gets added in by default but 6/1 0.5/12

Reply:
Good point. I'll just add a third case.